tests/testthat/test-check_dates.R

```diff
@@ -49,5 +49,13 @@ test_that("check end date after start date", {
   )
 })
 
+test_that("check end date after start date if end = start", {
+  start_date <- check_dates(c('1/1/21'))
+  end_date <- check_dates(c('1/1/21'))
+  expect_error(
```

Comment:
this should not error, right? aren't we expecting success here?

Reply:
oops, copy paste error. thanks for catching!